### ISSUE TITLE
Dose the code implement the targetd attack?

### ISSUE THREAD

**Issue by linsencc** [veralauee/DPatch#8]
hello~ the untargetd code I don't seem to see it, or I just miss it?

**Comment by alexlorenzo** [comment]
Hello did you finally find it the "trained DPATCH" ? @linsencc

**Comment by linsencc** [comment]
the code?

**Comment by alexlorenzo** [comment]
Yes, since I didn't find the code neither the `untargeted_DPATCH.npy`.
I train the model again, `python train.py`

**Comment by wmrenr** [comment]
> Yes, since I didn't find the code neither the `untargeted_DPATCH.npy`. I train the model again, `python train.py`

Have you solved it? I can't also find 'untargetd'_ DPATCH.npy'

**Comment by zhongguadepi** [comment]
> 你好，你找到untargeted_DPATCH.npy了吗，有的话麻烦私发一份451855925@qq.com   谢谢

**Comment by zhongguadepi** [comment]
> 是的，因为我既没有找到代码，也没有找到`untargeted_DPATCH.npy`. 我再次训练模型，`python train.py`
你好，你找到untargeted_DPATCH.npy  了吗，有的话，可以发一份吗451855925@qq.com

**Comment by zhongguadepi** [comment]
> 你解决了吗？我也找不到'untargetd'_ DPATCH.npy'

你解决了吗？我也找不到'untargetd'_ DPATCH.npy'

**Comment by srh20** [comment]
> > 你解决了吗？我也找不到'untargetd'_ DPATCH.npy'
> 
> 你解决了吗？我也找不到'untargetd'_ DPATCH.npy'

你好，请问你找到untargeted_DPATCH.npy 了吗，有的话，可以发一份吗[2848944634@qq.com]
